gprofiler/profilers/java.py

```diff
@@ -473,21 +473,24 @@ def _get_stop_cmd(self, with_output: bool) -> List[str]:
             + self._get_extra_ap_args()
         ]
 
+    def _read_ap_log(self) -> str:
+        if not os.path.exists(self._log_path_host):
```

Comment:
what if the log is created inside a container?

Reply:
`_log_path_host` includes the `/proc/pid/root`